src/components/DateRangePicker.jsx

```diff
@@ -238,6 +241,11 @@ class DateRangePicker extends React.PureComponent {
     });
   }
 
+  onDayPickerFocusOut(e) {
+    if (this.dayPickerContainer.contains(e.relatedTarget)) return;
```

Comment:
Just want to confirm that clicking on the month navigation, keyboard shortcuts, or any white space in the calendar does *not* close the picker, yeah?

Reply:
@majapw Hmm... Apparently it does? Not sure why, I'll look into it. I'd expect this contains check to work but I may need to get a ref to a parent container?

Reply:
@majapw I've pushed a fix - good find! I also fixed another instance of the same issue.